tools/code-analyzer/src/print.ts

```diff
@@ -114,9 +114,10 @@ export const printSchemaChange = (
 	version: string,
 	output: string,
 	log: ( s: string ) => void
-): void => {
+): Record< string, string > => {
+	const diff: Record< string, string > = {};
```

Comment:
This is a quick workaround to get the info I need from this function. Refactoring in future would be needed and I've raised an issue about it.